taskcluster/tc-cpp_tflite-ds-tests.sh

```diff
@@ -23,3 +23,5 @@ run_all_inference_tests
 run_multi_inference_tests
 
 run_cpp_only_inference_tests
+
+run_hotword_tests
```

Comment:
Can we also add coverage on `taskcluster/tc-android-ds-tests.sh` ?

Or add the `run_hotword_tests` code call from both `run_tflite_basic_inference_tests` and `run_basic_inference_tests` (but this might require a bit more work to implement the user-facing interface everywhere)

Reply:
added to `tc-android-ds-tests.sh` in fe7275e654c5e682a67043ed14c42c7d59b65c2d